### ISSUE TITLE
Noobie

### ISSUE THREAD

**Issue by Greatdanesrool** [Enterrados/AnimalJam-OpenBullet-Config#4]
Can someone help me out a little bit, I have installed countless wordlists, nine work, and I don't even know what a proxy is, lol. Any help/suggestions are appreciated!!

**Comment by uhvinia** [comment]
Where did u get the wordlists? I can help you if proxies if u want :)

**Comment by mortu1sm** [comment]
> Where did u get the wordlists? I can help you if proxies if u want :)

hey could you help with proxies and configs??

**Comment by mortu1sm** [comment]
> Can someone help me out a little bit, I have installed countless wordlists, nine work, and I don't even know what a proxy is, lol. Any help/suggestions are appreciated!!

what wordlists?? none of mine work..

**Comment by uhvinia** [comment]
> > Where did u get the wordlists? I can help you if proxies if u want :)
> 
> hey could you help with proxies and configs??

Of course!! I know a site to get some for free (its limited tho, but no need for ur cred card info)

**Comment by mortu1sm** [comment]
mainly need wordlists for the aj config i have everything else figured out:)

**Comment by Sizeable-Manchild** [comment]
I love how many skids there are in the issues section of this repo.

**Comment by KIRI2301** [comment]
> > > Where did u get the wordlists? I can help you if proxies if u want :)
> > 
> > 
> > hey could you help with proxies and configs??
> 
> Of course!! I know a site to get some for free (its limited tho, but no need for ur cred card info)

hey, can you send the wordlist please?
